pkg/operator/ceph/object/controller.go

```diff
@@ -236,50 +239,39 @@ func (r *ReconcileCephObjectStore) reconcile(request reconcile.Request) (reconci
 		r.clusterInfo.CephVersion = runningCephVersion
 		r.clusterInfo.Context = r.opManagerContext
 
-		if _, ok := r.objectStoreContexts[cephObjectStore.Name]; ok {
-			if r.objectStoreContexts[cephObjectStore.Name].internalCtx.Err() == nil {
-				// get the latest version of the object now that the health checker is stopped
```

Comment:
I don't believe this was ever actually necessary to stop the health checker to look for dependents. At worst, there could be an error checking that causes the reconcile to re-run, but I don't think that is possible unless we have multiple simultaneous reconciles in the future. Therefore, I remove the if/if check and un-indent the stuff below.